vendor/vendor.json

```diff
@@ -587,6 +629,72 @@
 			"revision": "efb555f6a4e9e5841f162d37bd0684fbf9cc4984",
 			"revisionTime": "2017-12-28T16:00:23Z"
 		},
+		{
+			"checksumSHA1": "YKIcl6SFm1Fc9UT4fptu34eqEC4=",
+			"path": "github.com/yunify/qingstor-sdk-go",
```

Comment:
In alpha version, we will support qingstor as repo address, but the protocol is s3.
In beta version, we will support other s3 protocol address, such as aws object storage. 
So in current alpha version, you need to visit qingstor with s3 sdk. These qingstor sdk should be deleted.